draft-ietf-keytrans-protocol.md

```diff
@@ -1229,15 +1511,18 @@ struct {
 
 If the request passes application-layer policy checks, the Transparency Log adds
 a new label-version pair to the prefix tree, followed by adding a new entry to
-the log tree with the associated value and updated prefix tree root. It returns
+the log tree with an updated timestamp and prefix tree root. It returns
```

Comment:
I think the draft must specify that clients must check that the timestamp the log shows them is sufficiently close to their submission. Otherwise, all that will ever be checked is monotonicity.

Reply:
Verifying that the rightmost timestamp is sufficiently recent is part of verifying "as if it were a SearchResponse for the most recent version of `label`"